package/skelcd-control-openSUSE.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Mon Mar 25 11:13:31 UTC 2019 - Dirk Mueller <dmueller@suse.com>
+
+- Fix update repository for ports (forward port of change that
+  went into Leap 15.0)
```

Comment:
Please add a bugzilla or FATE number (`bsc#....`, `FATE#....`), it's required by a Jenkins CI check, it would fail without that.

Reply:
done